nx-dev/nx-dev/project.json

```diff
@@ -59,7 +66,9 @@
       "outputs": ["nx-dev/nx-dev/public/documentation"],
       "options": {
         "commands": [
-          "echo \"🛑️ REPLACE ME WITH STANDARD BUILD COMMAND 🛑\"",
+          "echo \"🌁 GENERATE OG IMAGES COMMAND 🌁\"",
+          "nx run nx-dev:generate-og-images --skip-nx-cache",
```

Comment:
Why do we need to skip cache here?

Reply:
We shouldn't, I mean there is no need since this command is not in the `cacheableOperations`, that was a leftover from an earlier test. Will remove in the next PR 👍